src/main/java/com/sngular/annotation/processor/mapping/ByteMapping.java

```diff
@@ -24,12 +24,12 @@ public final String getFieldType() {
 
   @Override
   public final String getFunctionType() {
-    return "byteType";
+    return "integerType";
```

Comment:
Please, keep the right type

Reply:
If you check the documentation of pact you can see there is no byteType, that is why change to integerType that is the close one we got.

https://docs.pact.io/implementation_guides/jvm/consumer